notebooks/samples/pytorch/lightning/TrainingAndPredictionWithPyTorchLightning.ipynb

```diff
@@ -85,7 +85,7 @@
       "outputs": [],
```

Comment:
<p>Nit: It looks like the accelerator argument is only used when you have GPUs? I'm inferring that from the code, but it would be helpful to mention explicitly that this accelerator argument applies only when using GPUs (or whatever the condition is).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>